src/SpacetimeDBClient.cs

```diff
@@ -954,4 +1086,22 @@ public void FrameTick()
             }
         }
     }
+
+    internal struct UintAllocator
+    {
+        /// <summary>
+        /// Interlocked.Increment only works with ints.
```

Comment:
I was surprised as I'm pretty sure it works with `uint`, but looks like not in .NET Standard. Oh, Unity... https://learn.microsoft.com/en-us/dotnet/api/system.threading.interlocked.increment?view=net-9.0#system-threading-interlocked-increment(system-uint32@)